src/main/jbake/templates/index.ftl

```diff
@@ -5,55 +5,62 @@
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_inventory"><i class="fa fa-sitemap"></i></a>
-                        <h4>Inventory</h4>
-                        <p>Keep track of the monitored topology and all the monitored resource metadata. <a href="/overview/index.html#_inventory">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_metric_data_visualization"><i class="fa fa-bar-chart"></i></a>
-                        <h4>Metric Data Visualization</h4>
-                        <p>View your data and add charts where you need them most, including real time monitoring. <a href="/overview/index.html#_metric_data_visualization">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_application_performance_management"><i class="fa fa-briefcase"></i></a>
-                        <h4>Application Performance Management</h4>
-                        <p>Capture information about the application invocations being executed across your servers. <a href="/hawkular-apm/index.html">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="http://manageiq.org/"><img src="http://avatars0.githubusercontent.com/ManageIQ?&s=48" alt="ManageIQ logo"></img></a>
-                        <h4>Integration with ManageIQ</h4>
-                        <p>ManageIQ lets you manage container, virtual, private, and public cloud infrastructures. <a href="http://manageiq.org/">More...</a></p>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </section>
+</div>
+
+<div class="container">
+  <div class="row">
+    <h2>What is Hawkular?</h2>
+    <div class="paragraph">
+      <p>
+        Hawkular a hawk with a monocular. Hawks are known to have a very sharp vision and very good hunters, they can catch preys anticipating their movements at a very fast speed.
+        The analogy is our goal to be able to monitor and catch anomalies in fast pace environments.
+      </p>
+    </div>
+    <div class="paragraph">
+      <p>
+        The organization started around the end of 2014 as a successor of the <a href="http://www.rhq-project.org">RHQ Project</a>.
+        It’s a set of open source (Apache License v2) components targeted for monitoring solutions and is sponsored by <a href="http://www.redhat.com">Red Hat</a>.
+      </p>
+    </div>
+  </div>
+
+  <div class="row">
+    <h2>Mission Statement and Strategy</h2>
+    <div class="paragraph">
+      <p>
+        <strong>Mission Statement</strong>: Be the most popular open source monitoring solution.
```

Comment:
Who is Hawkular for?
Hawkular’s mission is to become the most popular open source monitoring solution.

The main goal of the Hawkular organization is provide tools to monitor, view, predict, and guide to make informed decisions about deployed applications and infrastructure.